### ISSUE TITLE
Ideas to make ebp more accessible

### ISSUE THREAD

**Issue by mydea** [cibernox/ember-power-select#1539]
Related to https://github.com/cibernox/ember-power-select/pull/1306, I think there are a few fundamental changes that should be made to make this addon more accessible. Currently, it fails e.g. ember-a11y-testing checks, sometimes in fundamental ways.

IMHO the most straightforward way to make the behavior of this addon more accessible would be:

1. Make the options in the single select a radio-button list
2. Move the search of the multi-select into the dropdown (same as in single select), to have no nested interactives in trigger
3. Make the options in multi select a list of checkboxes
4. Make the "x" remove functionality in multi-select trigger "optional" (to avoid nested interactives) - you can still remove it via the dropdown.

This is how this looks in our app, as reference:

![Screenshot from 2022-09-05 15-21-20](https://user-images.githubusercontent.com/2411343/188459918-fe90ac1d-b486-4cba-86e2-a8463aedd93b.png)

![Screenshot from 2022-09-05 15-21-46](https://user-images.githubusercontent.com/2411343/188459939-5a0cd820-ed19-4ab3-8ef1-56659d334fa1.png)

With such a setup, I was able to satisfy ember-a11y-testing with a relatively simple setup. It also means that keyboard support for selecting/unselecting items etc. works pretty much out of the box without any special handling.

However, doing anything in this direction would definitely be a breaking change, so not sure if there is any interest in actually pursuing something like this. If so, I'd be happy to write up a PR.

**Comment by mkszepp** [comment]
Just as note.. we did nr. 2 in #1864 and is released in v8.6

**Comment by mkszepp** [comment]
@RobbieTheWagner not directly working on this, but there is planned a new major (cause of `any` types), in which we can add breaking changes...

Nr. 4 is something which we can introduce without going breaking in current version...

I think all other points are breaking...

Can you tell me exactly the issues which are coming up with `ember-a11y-testing`?

**Comment by mkszepp** [comment]
btw. i think there is also not necessary to switch to checkboxes / radio buttons... because with aria tags we can do the same things (and i think we have the most parts already https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/)
